services/auth/oauth2.go

```diff
@@ -91,7 +92,18 @@ func (o *OAuth2) userIDFromToken(req *http.Request, store DataStore) int64 {
 	}
 	t, err := auth_model.GetAccessTokenBySHA(tokenSHA)
 	if err != nil {
-		if !auth_model.IsErrAccessTokenNotExist(err) && !auth_model.IsErrAccessTokenEmpty(err) {
+		if auth_model.IsErrAccessTokenNotExist(err) {
```

Comment:
`errors.Is(err, util.ErrNotExist)`?

Reply:
I don't think so, it's out of scope, we should do it in another PR.